better_words/db/seed.py

```diff
@@ -0,0 +1,80 @@
+from db import session
```

Comment:
This file needs to get run any time we add new words or change words, with `python -m better_words.db.seed`, maybe could set it up to just run every time we deploy